src/haproxy/conn.go

```diff
@@ -0,0 +1,126 @@
+package haproxy
+
+import (
+	"bufio"
+	"net"
+	"time"
+
+	proxyproto "github.com/pires/go-proxyproto"
+)
+
+// Conn is a `net.Conn` compatible struct that handles PROXY header checking.
```

Comment:
Go documentation does not support markdown - the backticks are rendered as is and should be removed.

Reply:
I did not know that, oops.

Fixing in the next commit.